docs/babel.config.js

```diff
@@ -19,6 +19,7 @@ const alias = {
   '@material-ui/icons': '../packages/material-ui-icons/src',
   '@material-ui/lab': '../packages/material-ui-lab/src',
   '@material-ui/styles': '../packages/material-ui-styles/src',
+  '@material-ui/styled-engine': '@emotion/styled',
```

Comment:
Should we make this default, so that initial setup would be zero config, and than do this only for `styled-components`? At this moment at least we can only change this and test styled-components implementation on the docs.